servicetalk-grpc-netty/src/test/java/io/servicetalk/grpc/netty/ProtocolCompatibilityTest.java

```diff
@@ -293,19 +307,53 @@ void serviceTalkBlockingToServiceTalkBlocking(final boolean ssl,
                                                   final boolean streaming,
                                                   final String compression)
             throws Exception {
-        final TestServerContext server = serviceTalkServerBlocking(ErrorMode.NONE, ssl, compression);
+        final TestServerContext server = serviceTalkServerBlocking(ErrorMode.NONE, ssl, compression, null);
         final BlockingCompatClient client = serviceTalkClient(server.listenAddress(), ssl, compression, null)
                 .asBlockingClient();
         testBlockingRequestResponse(client, server, streaming, compression);
     }
 
+    @ParameterizedTest(name = "{displayName} [{index}]: streaming={0} message={1}")
+    @MethodSource("statusMessageParams")
+    void serviceTalkToServiceTalkStatusMessage(final boolean streaming, final String message) throws Exception {
```

Comment:
I've opted to not also test compression and ssl with this since I figured it just makes the testing matrix very complex for not doing much on the task at hand. Streaming/non-streaming seems like it could make sense to this and just adds 1 more dimension.